src/components/expandable-text/expandable-text.js

```diff
@@ -86,7 +86,7 @@ const ExpandableText = withText({
   }
 
   return (
-    <div>
+    <div className={['expandable-text-container', isTextExpanded ? 'text-expanded' : ''].join(' ')}>
```

Comment:
why is this needed ? this would create a coupling with these texts to somewhere else, and it could break if it's changed in one of the repos